src/guide/command-line-options.md

```diff
@@ -135,10 +135,10 @@ This is a name of console output formatter. Possible values are: `dot`, `progres
 
 ### `--log-verbosity`
 
-The verbosity of the log file, `1` - debug mode (default) will add "Killed mutants" into log file and add additional information, `2` - normal mode will skip "Killed mutants" section in the log file, `3` - which will disable logging to files.
+The verbosity of the log file, `debug` - debug mode will add "Killed mutants" into log file and add additional information, `default` - normal mode (default) will skip "Killed mutants" section in the log file, `none` - which will disable logging to files.
```

Comment:
`default - normal mode (default) will`

imho, `(default)` is redundant here as we are describing `default` ;)